### DISCUSSION TITLE
How to edit e2sSample.all files.

### DISCUSSION THREAD

**Discussion by bangcorrupt** [bangcorrupt/hacktribe#29] [Q&A] [answered]
#27  gradientledge commented

> i have 2 programs to edit the .all files but are full of bugs, and every all file pops an error when i try to load it any suggestions? i pretty much just put the blank in there and edited it from the device.

**Comment by bangcorrupt** [comment, marked as answer]
I've always used [Oe2sSLE](https://github.com/JonathanTaquet/Oe2sSLE).  I've had the occasional crash, but no major issues.  Which editors are you using?

**Reply by gradientledge** [reply]
ive used that one if its the java one sample manager works but not as fast as being able to drop in one go i have my stuff in wavs of 16 48000hz but i even get errors with some of those, in the .all file exclusive to korg or is this a common format just renamed with an .all extension?

**Reply by bangcorrupt** [reply]
It starts with offsets to each block, then each sample block has the wav file followed by parameters specific to the electribe, like loop and slice points.  I haven't looked at it in detail.

**Reply by gradientledge** [reply]
oh that's interesting ill look into this this evening.

**Comment by BKLronin** [comment]
I see this with both programs on some samples too. Oe2sSLE seems a bit more picky but you can load complete folders and edit slice points etc.

**Reply by gradientledge** [reply]
i definitely tried it last night win 10 seems to crash anything like this when i feels like it so never know when but im going to try it again today and see

**Reply by BKLronin** [reply]
The electribe sample manager can get unresponsive while loading multiple files. Just be patient and dont touch anything.
